src/workspaces/NewWorkspaceModal/NewWorkspaceModal.ts

```diff
@@ -88,6 +88,7 @@ export interface NewWorkspaceModalProps {
   customMessage?: ReactNode;
   requiredAuthDomain?: string;
   requireEnhancedBucketLogging?: boolean;
+  showControlledAccessImportNotice?: boolean;
```

Comment:
Can you create a structure that contains all the import-specific options? When I was working with `NewWorkspaceModal`, the proliferation of input args was confusing to me… I had to see who was passing them in to understand their purpose. Perhaps something like:

```
interface ImportDataOptions {
  requireEnhancedBucketLogging;
  showControlledAccessNotice;
…
}

Reply:
Another thought… instead of adding another option, you could just make use of `customMessage`.

Reply:
> instead of adding another option, you could just make use of customMessage.

That's what I tried at first, but it only makes sense to show this when an Azure billing project is selected since GCP workspaces don't have policies.

Reply:
There is no other use case for those properties besides importing data. If a user is cloning a workspace, the properties are tied to the original workspace. If they are creating a new workspace from scratch, there is no reason to enforce those properties… they are either tied to the billing project selected or the additional options the user chooses in the dialog.

With the code as it is, it's difficult to reason about the combination of things. For instance, I had to think through if `cloneWorkspace` would ever be supplied at the same time as these options.

Reply:
That's the only place they're currently used, but conceptually, they're unrelated to importing data. Not all of them are overridden when cloning workspace. For example, `requiredAuthDomain` still applies in addition to the cloned workspace properties.

But it sounds to me like things could be simplified by separating components for creating a new workspace and cloning an existing workspace.

Reply:
I guess the code can remain as-is, though I think grouping would provide more context about why and how the options are used.

Reply:
You've convinced me about the `customMessage` prop. That will at least prevent adding a new prop here. We could also replace `workflowImport` with a combination of `customMessage` and `cloudPlatform`.

Reply:
Going to break this up into two PRs though... starting with #4523.